### PR TITLE
Added weeknotes to Feb 25

### PR DESCRIPTION
**Pull request by fwkendall**
Added weeknotes in markdown.

### CODE REVIEW

File: week-notes/2020-01-19.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+title: 2020-01-19
+parent: Week Notes
+---
+# BC Common Components (CoCo) Weeknote 1
+> ## 2020-01-19
+___
+
+This is the first of a series of weeknotes recapping what the BC Common Components (we’ve taken to calling it “CoCo”) team has done this week and what we plan to accomplish next week. I’ll save all of the weeknotes here.
+
+On Friday, I overhead Roop summarize the week best: “It feels like we need to design an ecosystem. It’s an incredible design challenge, but it’s not easy to figure out where to start.”  This week - the start of “Sprint 0”- was all about figuring out where to start.  We onboarded to the lab, began forming as a team, and began initial discovery work. We learned a lot and started shaping a few ideas about how we might rapidly add value to other tech teams in BC. We’ll further shape these ideas this coming week with the broader team (CGI and Exchange Lab), with the aim to get to a decision about the first components we take on in the next few days.
```
Comment (jpfournier) [inline week-notes/2020-01-19.md]: Change "overhead Roop" to "overheard one of our designers"

File: week-notes/2020-01-24.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# BC Common Components (CoCo) Weeknote 2:
+> ## 2020-01-24
+___
+
+This is the second post in a series of weeknotes recapping what the BC Common Components (we’ve taken to calling it “CoCo”) team has done this week and what we plan to accomplish next week. All previous weeknotes are saved here. There’s also now a CoCo mission control dashboard you can track [here](https://trello.com/b/vqqXYk3l/common-components-mission-control).
+
+## Done
+- Participated in a lunch and share with the digital community to learn about what others are building and how. We had a lively conversation about Common Components and associated pain points.
+- Attended ~5 sprint reviews hosted by other teams. Highlights were the [Health Gateway team](https://github.com/bcgov/healthgateway) and [Climate Action Secretariat](https://github.com/bcgov/cas-ggircs), both exemplary teams working on impactful products and building for reuse.
+- Interviewed several teams and product owners, including GDX, Climate Action, and Mines. My highlight was an interview Terry and I had with Jill Carruthers (Mines).
```
Comment (jpfournier) [inline week-notes/2020-01-24.md]: Remove reference to Terry and Jill. We can just pull that sentence out completely.

File: week-notes/2020-01-31.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- Bringing shaping groups together mid week to compare notes on parallel shaping work as shortlisted above. The bets include:
+  - Improving the developer site
+  - Identity Management
+  - Secure File Delivery
+  - Address validation
+  - UI Elements
+  - Boilerplate Final State
+- Stand up the project wiki, which will be hosted on Github pages
+
+Thanks,
```
Comment (jpfournier) [inline week-notes/2020-01-31.md]: Remove this and the "Michael and J-P" line below.

File: week-notes/2020-02-07.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- Finalizing and launch the project wiki.
+- Continue developing Launchpad.
+- Continue user research interviews with digital teams.
+- Sprint review session for Sprint 1.
+- Plan Sprint 2.
+- Firm up the shortlisted bets.
+
+Thanks,
+
+Michael and JP.
```
Comment (jpfournier) [inline week-notes/2020-02-07.md]: Remove reference to Michael and J-P

File: week-notes/2020-02-14.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  - Service Offering & Consumption Process
+- Continue shaping:
+  - Identity Management
+  - Secure File Transfer
+  - UI Library
+- New initiative to research and shape hit the table this week. Offline Mapping: provide program areas the ability to sync a map with BC Gov map data, go offline, add mapping data (boundaries of floods, crop hail damage areas, extent of invasive species infestations, active fire locations and paths, etc…), go back online and sync to BC gov map data
+ 
+Thanks,
+
+Michael and JP.
```
Comment (jpfournier) [inline week-notes/2020-02-14.md]: Remove reference to Rumon and J-P

File: week-notes/2020-02-23.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+- Prepare and schedule a next betting table meeting. 
+- Finalize the shaping of:
+  - Boilerplate “Launchpad” 3rd Iteration
+  - Common Components Library
+  - Service Offering & Consumption Process.
+- Kick off Sprint 3.
+ 
+Thanks,
+ 
+Michael and JP.
```
Comment (jpfournier) [inline week-notes/2020-02-23.md]: Remove reference to J-P and Michael.

**Comment by fwkendall** [comment]
Think I got all the changes requested, updated :)

**Pull request merged**